src/main/java/igrad/logic/commands/requirement/RequirementEditCommand.java

```diff
@@ -35,7 +35,7 @@
 
     public static final String MESSAGE_SUCCESS = "Edited Requirement: %1$s";
     public static final String MESSAGE_NOT_EDITED = "At least one field to edit must be provided.";
-    public static final String MESSAGE_REQUIREMENT_SAME_PARAMETERS = "Please change the name and/or the credits.";
+    public static final String MESSAGE_REQUIREMENT_SAME_PARAMETERS = "Please change the title and/or the credits.";
```

Comment:
sorry about this, wayne and i left it out, there's just too many to change even if we were careful 😂